graphene/root.crt

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This is just a DigiCert root certificate to run the example. In normal scenarios it probably will be provisioned using protected files feature.